css/index.css

```diff
@@ -0,0 +1,99 @@
+html {
+  /* Setting a base font size of 10px give us easier rem calculations
+       Info: 1rem === 10px, 1.5rem === 15px, 2rem === 20px and so forth
+     */
+  font-size: 10px;
+}
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Helvetica Neue", Arial, sans-serif;
+  font-size: 1.6rem;
+  line-height: 1.5;
+  text-align: center;
+  color: #333;
+  margin: 0;
+}
+
+h1 {
+  font-size: 4rem;
+  margin-bottom: 0;
+}
+
+@media (max-width: 460px) {
+  h1 {
+    font-size: 3.5rem;
+    line-height: 1.2;
+  }
+}
+
+h2 {
+  font-size: 3.25rem;
+}
+
+a {
+  color: #477ca7;
+}
+
+a:visited {
+  color: #74638f;
+}
+
+#main {
+  margin: 30px 8px;
+  padding: 15px;
+  border-radius: 5px;
+  background: #eee;
+  padding-left: 20px;
+}
+
+@media (max-width: 460px) {
+  #main {
+    margin: 0;
+  }
+}
```

Comment:
Good work on the CSS structure but I would highly recommend you leave comments, it will make your work easier to read. Kindly check the link provided; https://www.w3schools.com/css/css_comments.asp

Reply:
-  Kindly apply my first suggestion to this comment as well 👍🏽